docusaurus/docs/tutorial/01.md

```diff
@@ -3,213 +3,442 @@ id: 01
-__`Please note that this name is an example and you may change it with anything.`__
+```
+// yarn
+~$ yarn global add create-treats-app
+
+// npm
+~$ npm install -g create-treats-app
+
+~$ create-treats-app
+```
+
+A prompt asking project's name, version, description, and usage of Typescript will be shown. Please fill it all or just enter if you want it to be filled with the default values. Before we start, let's assume we want to create a page named `todo`.  
+
+__Please note that this name is an example and you may change it with anything.__
 
 Now, let's get started:
 
-1. Create new folder named __`mypage`__, containing __`index.js`__ and __`mypage.css`__. Your folder should look like this:  
+## Creating Component
+Before creating our page and to improve our understanding in creating components using Treats, let's make our list in separate component.  
+
+1. Create a folder `component` under `src`, which contain `todo-list` folder inside it. In `todo-list` create __`index.js`__, __`todo-list.js`__, and __`todo-list.css`__ (optional). Your folder structure should be like following:  
+
 ```
-mypage  
-|  
-|-- index.js  
-|-- mypage.css  
+src
+|-- _locale
+|-- _route
+|-- component
+    |-- todo-list
+        |-- index.js
+        |-- todo-list.css
+        |-- todo-list.js
+|-- page
 ```
 
+2. Write our `index.js` file:
 ```
-/* index.js */
-import MyPage from "./mypage";
+/* src/component/todo-list/index.js */
+
+import TodoList from "./todo-list";
+
+export default TodoList;
+```
+  
+At this point, you may be wondering why we need `index.js` that only import the component and export it. To put it simple, it will make our component import easier. Here are the difference:  
 
-export default MyPage;
 ```
+//with index.js
+import Todo from "./src/component/todo"
 
+//without index.js
+import Todo from "./src/component/todo/todo"
+```  
 
-2. Write __`index.js`__ for your page
+3. Write our `todo-list.js` file:
 ```
-/* ./src/page/mypage/index.js */
+/* src/component/todo-list/todo-list.js */
 
 import React from "react";
-import Helmet from "@treats/helmet";
-import AsyncComponent from "@treats/component/async-component";
-import style from "./mypage.css";
-
-const pStyle = {
-    fontSize: '12px',
-    textAlign: 'center'
-  };
-  
-const MyPage = () =>(
-    <div className={style.divStyles}>
-        <h1>My Page</h1>
-        <p style={pStyle}>
-            Lorem Ipsum is simply dummy text of
-            the printing and typesetting industry.
-        </p>
-    </div>
+
+const TodoList = ({items}) => (
+    <ul>
+        {items && items.map((item, index) => (
+            <li key={index}>
+                {item}
+            </li>
+        ))}
+    </ul>
 );
 
+export default TodoList;
+```
+
+4. (Optional) Style your component in `todo-list.css`.
+
+### The easier way
+Treats provide component boilerplate [generators][Generator]. So you will only need to type some commands and jump to step 3. Here are the commands:
```

Comment:
Treats can generate component boilerplate using [generators][Generator], so we can jump to 3rd step.

rephrase it this way? to give more clarity